eslint.config.js

```diff
@@ -104,4 +105,11 @@ export default defineConfig([
 		language: "json/json",
 		extends: ["json/recommended"],
 	},
+	{
+		name: "css/css",
+		files: ["**/*.css"],
+		language: "css/css",
+		plugins: { css },
+		extends: ["css/recommended"],
+	},
```

Comment:
```suggestion
        // This CSS configuration is mainly used to validate the `test.css` file for local testing.
	{
		name: "css/css",
		plugins: { css },
		files: ["**/*.css"],
		language: "css/css",
		extends: ["css/recommended"],
	},
```

Can we add a comment like above to ensure not to be removed in the future by mistake?

Also, as a non-blocking request, I think it would be nice if the property keys were ordered `name` -> `plugins`, like in the JSON configuration.